CHANGELOG.md

```diff
@@ -7,13 +7,14 @@
  - Add Nooddorpen subsidy types via extractor (DGS-166)
 ### General
 #### Frontend
- - Bump frontend to `v0.90.3` (DL-5709): https://github.com/lblod/frontend-loket/blob/development/CHANGELOG.md#v0903-2024-02-28
+ - Bump frontend to `v0.90.3` (DL-5708): https://github.com/lblod/frontend-loket/blob/development/CHANGELOG.md#v0903-2024-02-28
 #### Backend
  - Bump `migrations` to `v0.9.0`
+ - Bump `mock-login` to `v0.4.0` (DL-5709)
 ### Deploy Notes
- - remove the loket v0.90.3 image override from the docker-compose.override.yml
-#### Docker commands
- - `drc up -d migrations`
+ - Remove the frontend `v0.90.3` image override from `docker-compose.override.yml`
+#### Docker Commands
+ - `drc up -d migrations mocklogin`
```

Comment:
I think the version also needs to be bumped in the docker-compose.override.yml file, no? I think the .dev.yml version is only used in local setups?

Reply:
Yes, you're right; I missed this one. I'll update the deploy notes section to cover this.